invenio_records_marc21/system/service.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+#
+# Copyright (C) 2020-2021 CERN.
+# Copyright (C) 2020 Northwestern University.
+# Copyright (C) 2021 TU Wien.
+# Copyright (C) 2021 Graz University of Technology.
+#
+# Invenio-Records-Marc21 is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Marc21 Record Resource."""
+
+
+from flask_resources import route
+
+from . import config
+from .models import Marc21Templates
+
+
+#
+# Records
+#
+class Marc21TemplateService:
+    """Marc21 template resource."""
+
+    config_name = "MARC21_TEMPLATE_CONFIG"
+    default_config = config.Marc21TemplateConfig
+    template_cls = Marc21Templates
+
+    def __init__(self, config):
+        """Constructor for Marc21TemplateService."""
+        self.config = config
+
+    def create(self, name, data):
+        """Create a template for a new record."""
+        return self._create(self.template_cls, name, data)
+
+    def _create(self, template_cls, name, data):
```

Comment:
it is not necessary to create a new method only for that small thing, especially if nothing else happens in the calling function